modules/core/src/effects/lighting/camera-light.js

```diff
@@ -0,0 +1,17 @@
+import PointLight from './point-light';
+import {getUniformsFromViewport} from '../../shaderlib/project/viewport-uniforms';
+
+export default class CameraLight extends PointLight {
+  getProjectedLight(layer) {
```

Comment:
I would use named parameters (`{layer}`) in case we need to add additional contextual info in the future.